### PR TITLE
#171967419 Fix In-App notifications and get one trip request feature.

### PR DESCRIPTION
**Pull request by RusimbiPatrick**
### What does this PR do?

- Notification returns the createdAt date.
- All notifications are displayed on the static file
- Both the user and his manager are able to view a trip request

### How should this be manually tested?

- 
- Go to` /static` and log in with receiver ID and host
- Trigger a notification
- checkout the static file for notifications on the front-end and in the console

### Implementation plan
N/A

### Stories associated with PR

[#171967419](https://www.pivotaltracker.com/story/show/171967419)

### Screenshots if any
Screen Shot 2020-03-25 at 04 39 52
Screen Shot 2020-03-25 at 04 32 46

### CODE REVIEW

**Review by AbdoulNuru — COMMENTED** [review]
I have concerns, why are you changing a lot of things? I can see you have changes every method in notificationController and commentsController?

**Comment by blaise82** [comment]
> I have concerns, why are you changing a lot of things? I can see you have changes every method in notificationController and commentsController?

this is not good

**Comment by RusimbiPatrick** [comment]
> I have concerns, why are you changing a lot of things? I can see you have changes every method in notificationController and commentsController?

The purpose of this  PR was to fix things. And as you have seen in pre-demo all of them are working. In the comments controller, there was no way to get the comment Id so to delete it.

**Comment by RusimbiPatrick** [comment]
> > I have concerns, why are you changing a lot of things? I can see you have changes every method in notificationController and commentsController?
> 
> this is not good

Fixing things?

**Review by blaise82 — APPROVED** [review]

File: public/index.html [hunk 1]
```diff
@@ -1,82 +1,99 @@
...
+                    $("#notifications").append(`<li><a href=${link} target="_blank">${body}</li>`);
+                    console.log(notification);
+                }
+            });
         }
     </script>
 </body>
-</html>
+
+</html>
```
Comment (AbdoulNuru) [inline public/index.html]: new line

File: src/controllers/commentsController.js [hunk 2]
```diff
@@ -10,71 +10,75 @@ import notificationsController from './notification';
...
-      });
-      if (!found) {
-        return errorResponse(res, 403, setLanguage(preferedLang).__('commentSorry'));
-      }
-      const newComment = await Comments.create({
-        id: uuid(),
-        userID: id,
-        requestId,
-        comment
-      });
```
Comment (AbdoulNuru) [inline src/controllers/commentsController.js:46]: @RusimbiP for example you copied this block of code, deleted it and pasted it below but there is no single change both blocks are identical, so I was asking why you did that.

**Review by hakoemmy — APPROVED** [review]

File: src/controllers/commentsController.js (hunk 2 above)
Comment (RusimbiPatrick) [reply src/controllers/commentsController.js:46]: Ooooh, It is prettier

Comment (RusimbiPatrick) [reply src/controllers/commentsController.js:46]: The vscode extension

**Review by AbdoulNuru — APPROVED** [review]

**Pull request merged**